examples/README.md

```diff
@@ -16,7 +16,9 @@ relevant code.
 
 
 ## Digging deeper
-- [Searching in a transformed parameter space](./optimisation/transformed-parameters.ipynb)
+- [Optimisation in a transformed parameter space with a model wrapper](./optimisation/transformed-parameters-model-wrapper.ipynb)
```

Comment:
The do-it-yourself example can go, if this works

Reply:
We could maybe have one folder that shows some "hack around Pints", then we could put this in, as well as some examples showing using Pints for non-time series problems etc.